versions/l-/libbson.json

```diff
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "7481b6789873a1e19429de28f48b4eb3dd95569a",
```

Comment:
Needs a new port version

Reply:
These changes are based on my update PR, these two ports have update to new version 1.30.3, so these changes should use the same value of port version 1.30.3.